### PR TITLE
server 4.0.1 release docs

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Docs to release new version of server.

# Reasons
https://circleci.atlassian.net/browse/DOCTEAM-757?atlOrigin=eyJpIjoiYTQ3ZTQyOTNmZWJiNDAxNThlMDZiODRmMzU4NDUzYTMiLCJwIjoiaiJ9

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/overview/release-notes.adoc [hunk 1]
```diff
@@ -11,16 +11,33 @@ version:
...
-[#upgrade-notes]
-=== Upgrade notes
+== Migrate from v3.x to v4.x
 
 For existing customers interested in migrating from v3.x to v4.x, or v2.x to v4.x contact your customer success manager. Server v4.x will receive monthly patch releases and quarterly feature releases.
 
+Find information about migration in the xref:server/installation/migrate-from-server-3-to-server-4.adoc[Migrate From Server v3 to v4 Guide].
+
+== Upgrade
+For upgrade steps see the xref:server/installation/upgrade-server-4.adoc[Upgrade Server v4 Guide].
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/overview/release-notes.adoc:28]:
```suggestion
For upgrade steps see the link:docs/installation/upgrade-server-4[Upgrade Server v4 Guide]. 
```

File: jekyll/_cci2/server/overview/release-notes.adoc [hunk 2]
```diff
@@ -11,16 +11,33 @@ version:
...
+* Installation processes can use artifacts pulled from customer-managed registries. 
+* Improved use of Kubernetes secrets and removes the requirement to grant permissions and network access to third-party tools. 
 
-[#upgrade-notes]
-=== Upgrade notes
+== Migrate from v3.x to v4.x
 
 For existing customers interested in migrating from v3.x to v4.x, or v2.x to v4.x contact your customer success manager. Server v4.x will receive monthly patch releases and quarterly feature releases.
 
+Find information about migration in the xref:server/installation/migrate-from-server-3-to-server-4.adoc[Migrate From Server v3 to v4 Guide].
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/overview/release-notes.adoc:25]:
```suggestion
Find information about migration in the link:docs/server/installation/migrate-from-server-3-to-server-4[Migrate From Server v3 to v4 Guide].
```

File: jekyll/_cci2/server/overview/release-notes.adoc [hunk 3]
```diff
@@ -11,16 +11,33 @@ version:
...
+
+== Upgrade
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/overview/release-notes.adoc:27]:
```suggestion
[#upgrade]
== Upgrade
```

File: jekyll/_cci2/server/overview/release-notes.adoc [hunk 4]
```diff
@@ -11,16 +11,33 @@ version:
 :toc: macro
 :toc-title:
 
-[#release-4]
-== Release 4.0.0
+Server v4.x offers the following:
 
-Server v4.x offers greatly improved security handling, installation, and update processes. Server v4.x is installed using helm charts and images that can be pulled ahead of time to comply with your security processes. Installation processes can also use artifacts pulled from customer-managed registries. Server v4.x makes improved use of Kubernetes secrets and removes the requirement to grant permissions and network access to third-party tools. 
+* Greatly improved security handling, installation, and update processes. 
+* Installed using helm charts and images that can be pulled ahead of time to comply with your security processes.
```
Comment (devinmarieb) [inline jekyll/_cci2/server/overview/release-notes.adoc:17]: Should Helm be capitalized?

**Review by devinmarieb — APPROVED** [review]

File: jekyll/_cci2/server/overview/release-notes.adoc [hunk 5]
```diff
@@ -11,16 +11,34 @@ version:
...
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/overview/release-notes.adoc:17]:
```suggestion
* Installed using Helm charts and images that can be pulled ahead of time to comply with your security processes. 
```

File: jekyll/_cci2/server/overview/release-notes.adoc (hunk 4 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server/overview/release-notes.adoc]: Yes @devinmarieb thank you!!

**Pull request merged**